homeworks/zlinszkygyorgy/7_extract_transform/settings.py

```diff
@@ -0,0 +1,10 @@
+import os
+import dotenv
+from dotenv import load_dotenv
+load_dotenv()
+
+#API_KEY
+COINGECKO_API_KEY = os.environ.get("CG-YDCCtb4GxTFNWjQhYkYv1K1Z")
```

Comment:
ezzel itt az a gond, hogy pont az api key-t nem kéne egy .py file-ban tárolnunk. 
Nekem (és senki másnak) nem lenne szabad ezt az api key-t itt látni.
Tárold ezt el kérlek egy .env file-ban, és az ottani változó nevére hivatkozz, ahogy órán is csináltuk.

Reply:
Szia, javítottam.